src/binaryutils.cpp

```diff
@@ -297,6 +298,18 @@ InferenceEngine::Blob::Ptr createBlobFromMats(const std::vector<cv::Mat>& images
     return blob;
 }
 
+ov::runtime::Tensor createBlobFromMats_2(const std::vector<cv::Mat>& images, const std::shared_ptr<TensorInfo>& tensorInfo, bool isPipeline) {
```

Comment:
maybe createTensorFromMats?

Reply:
Renaming is planned as part of a cleanup. Left as is for now to avoid conflicts